### PR TITLE
events/m3: Add content for evaluation and next-steps sections

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

File: website/docs/events/m3-slides/.slides/evaluation.md [hunk 1]
```diff
@@ -4,12 +4,71 @@
...
+## Evaluation
+
+- Student feedback
+- Lab Assistant Feedback
+- Grades
+- Contributions
+
+---
+
+## Student Evaluation
```
Comment (teodutu) [inline website/docs/events/m3-slides/.slides/evaluation.md:16]: Maybe it's not to our advantage, but I think you should mention the amount of students who submitted responses to the feedback form. All percentagee are divisible by 5, which makes them look kibda sus.

File: website/docs/events/m3-slides/.slides/next-steps.md [hunk 2]
```diff
@@ -2,3 +2,12 @@
...
 ## Next Steps
 
+----
+
+### Next Steps
+
+- Solve all issues
+- Develop new assignments
+- Reiterate content
+- Smiley Coin Integration
```
Comment (teodutu) [inline website/docs/events/m3-slides/.slides/next-steps.md:12]: I would develop on this a bit. Mention Mihai's recent progress.

File: website/docs/events/m3-slides/.slides/next-steps.md [hunk 3]
```diff
@@ -2,3 +2,12 @@
...
+- Quiz Interface
```
Comment (teodutu) [inline website/docs/events/m3-slides/.slides/next-steps.md:13]: Maybe use DSS/PCLP4 as an example?

**Review by teodutu — CHANGES_REQUESTED** [review]
I am on my phone so I can't build the slides. I hope all images fit within tge screen. I wrote anything beyond this concern as inline comments.

File: website/docs/events/m3-slides/.slides/evaluation.md (hunk 1 above)
Comment (RazvanN7) [reply website/docs/events/m3-slides/.slides/evaluation.md:16]: If someone asks, I'll tell.

File: website/docs/events/m3-slides/.slides/next-steps.md (hunk 2 above)
Comment (RazvanN7) [reply website/docs/events/m3-slides/.slides/next-steps.md:12]: Yes, with words.

File: website/docs/events/m3-slides/.slides/next-steps.md (hunk 3 above)
Comment (RazvanN7) [reply website/docs/events/m3-slides/.slides/next-steps.md:13]: With words.

**Comment by RazvanN7** [comment]
@teodutu What exactly is blocking this from being merged?

File: website/docs/events/m3-slides/.slides/evaluation.md [hunk 4]
```diff
@@ -4,12 +4,71 @@
...
+
 ---
 
+## Grades
+
+----
+
 ### Grades
+
+![Average Grade Per Year](avg-grade.png)
```
Comment (teodutu) [inline website/docs/events/m3-slides/.slides/evaluation.md]:
```suggestion
![Average Grade Per Year](media/avg-grade.png)
```

**Review by teodutu — CHANGES_REQUESTED** [review]
In the meantime, I've returned to my desk and built the slides. They look fine except for the broken link I highlighted. Fix that and then this can be merged.

**Review by teodutu — APPROVED** [review]

**Pull request merged**